toolchain/parse/state.def

```diff
@@ -1015,33 +1020,33 @@ CARBON_PARSE_STATE(StatementWhileBlockFinish)
 
 // Handles parsing after the declaration scope of a type.
 //
-// class/interface/constraint { ... }
-//                                  ^
+// class/impl/interface/constraint { ... }
```

Comment:
This is starting to feel like it should just be `<introducer>`

Reply:
Maybe, but this makes it easier to search for `class` or `interface` and find the relevant states.